apps/communications/contacts/js/views/settings.js

```diff
@@ -960,6 +980,13 @@ contacts.Settings = (function() {
 
   var checkNoContacts = function checkNoContacts() {
     var exportButton = exportContacts.firstElementChild;
+
+    var enableButtons = function() {
+      exportButton.removeAttribute('disabled');
+      bulkDeleteButton.removeAttribute('disabled');
+      setICEButton.removeAttribute('disabled');
+    };
+
     var req = navigator.mozContacts.getCount();
```

Comment:
I would suggest we do something like

var operations = [
  getNumContacts(),
  getNumFbContacts()
]

Promise.all(operations).then(function(results) {
})

and taking into account the results we enable disable the export, bulk and setICE buttons